spring-cloud-stream-module-launcher/README.md

```diff
@@ -0,0 +1,91 @@
+# Module Launcher
+
+The Module Launcher provides a single entry point that bootstraps module JARs located in a home directory. A single Docker image that contains such a directory of module JARs can then be used to launch any of those JARs based on an environment variable. When running standalone, a system property may be used instead of an environment variable, so that multiple instances of the Module Launcher may run on a single machine. The following examples demonstrate running the modules for the *ticktock* stream (`time | log`).
+
+## Prerequisites
+
+1: clone and build the spring-cloud-stream project:
+
+````
+git clone https://github.com/spring-cloud/spring-cloud-stream.git
+cd spring-cloud-stream
+mvn -s .settings.xml package
+cd ..
+````
+
+2: copy the spring-cloud-stream source and sink sample JARs to `/opt/spring/modules`:
+
+````
+mkdir -p /opt/spring/modules
+cp spring-cloud-stream/spring-cloud-stream-samples/source/target/spring-cloud-stream-sample-source-1.0.0.BUILD-SNAPSHOT-exec.jar /opt/spring/modules/time.jar
+cp spring-cloud-stream/spring-cloud-stream-samples/sink/target/spring-cloud-stream-sample-sink-1.0.0.BUILD-SNAPSHOT-exec.jar /opt/spring/modules/log.jar
+````
+
+3: start redis locally via `redis-server` (optionally start `redis-cli` and use the `MONITOR` command to watch activity)
+
```

Comment:
May want to put a note:   
**Note:* redis.conf (on OSX it is found here:  /usr/local/etc/redis.conf) may need to be updated to set the binding to an address other than 127.0.0.1 else the docker instances will fail to connect.  For example: `bind 0.0.0.0`

To test this I did a brew uninstall redis / brew install redis.  The new redis.conf had the binding set to 127.0.0.1 and the modules failed to connect.